python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -96,6 +96,10 @@ class PsfexPsfDeterminerConfig(measAlg.BasePsfDeterminerConfig):
         doc="Should PSFEX be permitted to recentroid PSF candidates?",
         default=False,
     )
+    photometricNormField = pexConfig.Field[str](
```

Comment:
Lets call this `PhotometricFluxField` instead, so it more closely matches the name of the field that is being overridden.